README.md

```diff
@@ -296,7 +296,7 @@ These goals motivated the following decisions:
   This is the introductory paragraph, with
   an identifier `introduction`.
 
-                             {.important color="blue" #heading}
+           {.important color="blue" #heading}
   ## heading
```

Comment:
I'd prefer the attribute to start a bit further to the right, so at least it doesn't overlap the heading text.
This is just to show that you can use a right-justified attribute if you want, to make it less liable to be confused with content text.

The other change is good, thanks!

Reply:
Ah, interesting. I didn't get that impression from the code sample. I'll make the change. I assume a single space will be sufficient, but let me know if you prefer two or more.